<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberCycleEditBase</class>
 <widget class="QWidget" name="NumberCycleEditBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>279</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="mHeader">
     <property name="text">
      <string>&lt;h3&gt;Edit Number Cycles&lt;/h3&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Number Cycle Details</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="mCounterEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="mIdTemplEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>&amp;Number cycle:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>mNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1_2_3">
        <property name="text">
         <string>&amp;Counter:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>mCounterEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="mHelpLabel">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel1_2_3_2">
        <property name="text">
         <string>Example Id:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel1_2_2">
        <property name="text">
         <string>ident &amp;Template:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>mIdTemplEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mNameEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="mExampleId">
        <property name="text">
         <string>example</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>&amp;Select a number cycle and edit the details on the right:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="buddy">
        <cstring>mCycleListBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QListWidget" name="mCycleListBox">
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <spacer name="spacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>37</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mPbAdd">
           <property name="toolTip">
            <string>Click to add a new document type to the list.</string>
           </property>
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mPbRemove">
           <property name="toolTip">
            <string>click to remove the current document type</string>
           </property>
           <property name="text">
            <string>remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
